src/Button/Button.tsx

```diff
@@ -361,37 +403,34 @@ export const Button = React.forwardRef<HTMLElement, Props>(
         {({ cx, css }) => {
           const propsToPass = mergeProps(
             // Omit `onClick` from `otherProps` beacuse we'll be conditionally
-            // calling it in the `onClick` handler depending on the `disabled`
-            // prop. Also exclude `className` beacuse we'll be combining it on
+            // calling it in the `onClick` handler depending on `mergedProps.disabled`.
+            // Also exclude `className` beacuse we'll be combining it on
             // our own with `cx`. This is necessary because `cx` allows for
             // emotion styles to be logically overwritten.
-            omit(mergedProps, "className", "onClick"),
+            omit(
+              mergedProps,
+              "className",
+              "onClick",
+              // If we're overriding the default disabled behavior, then strip
+              // it out from the props we'll pass to the element.
+              overrideDisabledBehavior ? "disabled" : "",
```

Comment:
Is there an aria prop we should/could pass to make sure the button is still semantically disabled?

Reply:
Only kind-of. We can use `aria-disabled`, but this will only provide special behavior for screen readers. I'll add it.

Reply:
ooh nice, thats probably where it matters most so thats perfect :)